lib/Browser.js

```diff
@@ -172,6 +179,48 @@ class Browser extends EventEmitter {
     return page;
   }
 
+  /**
+   * @param {?Protocol.Target.BrowserContextID} contextId
+   * @param {string} origin
+   * @param {!Array<string>} permissions
+   */
+  async _overridePermissions(contextId, origin, permissions) {
```

Comment:
I feel like this code should be in the browser context. That way if we ever add a helper method for the default browser context on the browser, it won’t conflict with the internal method.

Reply:
done.